txstratum/cli.py

```diff
@@ -51,16 +55,37 @@ def execute(args: Namespace) -> None:
     from txstratum.utils import start_logging
 
     # Configure log.
+    log_config_file = args.log_config or DEFAULT_LOGGING_CONFIG_FILE
+
     start_logging()
-    if os.path.exists(args.log_config):
-        logging.config.fileConfig(args.log_config)
+    if args.json_logs:
+        if args.log_config:
+            logger.error('--json-logs and --log-config are mutually exclusive')
+            exit(1)
```

Comment:
I guess this works, but I was thinking of using a `mutually_exclusive_group` like we use on hathor-core for example:

```python
        storage = parser.add_mutually_exclusive_group()
        storage.add_argument('--rocksdb-storage', action='store_true', help='Use RocksDB storage backend (default)')
        storage.add_argument('--memory-storage', action='store_true', help='Do not use any storage')
        storage.add_argument('--json-storage', action='store_true', help='Use legacy JSON storage (not recommended)')
```

Reply:
Yeah, it's better. Done in https://github.com/HathorNetwork/tx-mining-service/pull/45/commits/23895a29822f46420f5f385ff46e4d2a52fac9e0